# Standalone mode.
is_alone: true

robot_name: '/test_robot'
app_from_source_directory: '/home/jihoonl/ros/groovy/turtlebot/turtlebot_apps/turtlebot_core_apps'
app_store_url: []
platform_info: 'linux.ros.*'

white_list: ["concert_master"]
black_list: ["pirate_master"]

remote_name: "gateway1"
